openverse_api/catalog/settings.py

```diff
@@ -322,3 +324,17 @@
 
 # Whether to boost results by authority and popularity
 USE_RANK_FEATURES = config("USE_RANK_FEATURES", default=True, cast=bool)
+
+SENTRY_DSN = config(
+    "SENTRY_DSN",
+    default="https://08f4706d16004f57bcd37eb907bfc2e7@o787041.ingest.sentry.io/6107216",
```

Comment:
This is our sentry URL, right? Do we want to have this as the default in-code, or rather just have it as a config variable? It feels weird to have this be codified in our codebase when other people might spin this up or work on it, but perhaps it's an unnecessary concern.

Reply:
I feel that instead of a default, the fallback behaviour should be to turn off Sentry when the variable is not supplied.